bedrock/externalpages/templates/externalpages/pocket/firefox/new-tab-learn-more.html

```diff
@@ -0,0 +1,97 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "externalpages/pocket/base.html" %}
+
+{% from "macros-protocol.html" import split with context %}
+
+{% block pocket_css %}
+ {{ css_bundle('pocket-new-tab') }}
+{% endblock %}
+
+{% block content %}
+<header class="new-tab-header">
+    <div class="mzp-c-logo mzp-t-logo-xs mzp-t-product-family">Firefox</div>
+    <span>|</span>
+    <div class="mzp-c-logo mzp-t-logo-xs mzp-t-product-pocket">Pocket</div>
+    <p>Pocket is part of the Firefox family</p>
+</header>
+<section>
+    {% call split(
+        block_class='mzp-l-split-center-on-sm-md new-tab-section top',
+        media_class='mzp-l-split-h-center',
+        image_url='img/externalpages/pocket/new-tab/pocket-list.png',
+        )%}
+        <p class="section-top-header-lede">POCKET FOR FIREFOX</p>
+        <h2 class="new-tab-section-header top">Build your personal library of fascinating reads.</h1>
+        <p class="new-tab-section-body top">Included inside Firefox, the <span class="pocket-logo-inline">Pocket Logo</span> button lets you save articles from across the web and read them in a quiet, private space.</p>
+        <a class="new-tab-pocket-cta"><span class="mzp-c-logo mzp-t-logo-sm mzp-t-product-family">Firefox</span>Activate Pocket in Firefox</a>
+        <a class="sub-cta">More ways to sign up</a>
```

Comment:
These links are both missing `href`s?

Should the first link be a `pocket_fxa_button`? https://bedrock.readthedocs.io/en/latest/firefox-accounts.html#pocket-links

Reply:
Also the first link would be read aloud as "FirefoxActivate Pocket in Firefox". The logo seems presentational and complimentary to "Activate Pocket in Firefox", so the first "Firefox" is probably superfluous?